pkg/kubelet/network/exec/exec.go

```diff
@@ -29,6 +29,9 @@ limitations under the License.
 //         - setup, called after the infra container of a pod is
 //                created, but before other containers of the pod are created
 //         - teardown, called before the pod infra container is killed
+//         - status, called at regular intervals and is supposed to return a json
```

Comment:
I'm OK with it, just wanted to understand.  I'm a little concerned about overhead, but let's just keep an eye on it.